src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/integration/wiremock/PrisonApiMockServer.kt

```diff
@@ -281,15 +281,15 @@ class PrisonApiMockServer : WireMockServer(WIREMOCK_PORT) {
           .withBody(
             """
               {
-                "bookingId": 1234134,
+                "bookingId": $bookingId,
```

Comment:
since the requested `bookingId` can vary, it makes sense to respond with a matching one!
NB: no tests relied on past (arguably incorrect) functionality